app/models/topic.rb

```diff
@@ -1632,6 +1632,36 @@ def access_topic_via_group
       .first
   end
 
+  def incoming_email_addresses(group: nil, received_before: Time.zone.now)
+    email_addresses = group.present? ? Set[group.email_username] : Set.new
```

Comment:
Maybe I miss something, but is there any reason to add `group.email_username` to `email_addresses` if you are going to remove it anyway a few lines down?

```suggestion
    email_addresses = Set.new
```

Reply:
This is a good point, I only did this because that is what was there originally, will remove.